### PR TITLE
Add feature_rds and rds_articles

### PR DESCRIPTION
**Pull request by giorgiosironi**
Depends on https://github.com/elifesciences/builder-configuration/pull/119

Also fixes `aws.queue_name` in `config.php`, which was wrong in `dev`.

### CODE REVIEW

File: salt/search/config/srv-search-config.php [hunk 1]
```diff
@@ -14,11 +14,25 @@
...
+        {% endif %}
         'credential_file' => true,
         'region' => 'us-east-1',
         {% if pillar.search.aws.endpoint %}
         'endpoint' => '{{ pillar.search.aws.endpoint }}',
         {% endif %}
     ],
+    'feature_rds' => {% if pillar.search.feature_rds %}true{% else %}false {% endif %},
+    'rds_articles' => [
+		{% for id, values in pillar.search.rds_articles.items() %}
```
Comment (giorgiosironi) [inline salt/search/config/srv-search-config.php]: how did the tabs get in?

**Comment by giorgiosironi** [comment]
`builder-configuration` is there so would merge when green.

File: salt/pillar/search.sls [hunk 2]
```diff
@@ -24,6 +24,10 @@ search:
     ttl: 0
     rate_limit_minimum_page: 2
 
+    feature_rds: true
+    {% import_yaml "rds-articles.yaml" as rds_articles %}
+    rds_articles: {{ rds_articles|yaml }}
```
Comment (thewilkybarkid) [inline salt/pillar/search.sls]: Can remove the import now?

Comment (giorgiosironi) [reply salt/pillar/search.sls]: It reflects what is going on in `builder-configuration` to keep it tested in case of refactoring.

**Pull request merged**